alpine/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM alpine:3.3
 
+RUN apk --no-cache add \
+	ca-certificates
+
```

Comment:
Micro optimization tip:

Each `apk add --no-cache` will fetch the APKINDEX from network and throw it away after use. It means that we will fetch the same APKINDEX over the network 3 times due to the `apk add --no-cache --virtual .build-deps ...` and `apk add --no-cache --virtual .irssi-rundeps ...` below.

To avoid that we could add the `ca-certificates` and `perl-libwww` to the `.build-deps` and add them again to the `.irssi-rundeps` but exclude `--no-cache` option there. Eg.

````diff
diff --git a/alpine/Dockerfile b/alpine/Dockerfile
index bdfd967..15cea8f 100644
--- a/alpine/Dockerfile
+++ b/alpine/Dockerfile
@@ -1,8 +1,5 @@
 FROM alpine:3.3
 
-RUN apk --no-cache add \
-       ca-certificates
-
 ENV HOME /home/user
 RUN adduser -u 1001 -D user \
        && mkdir -p $HOME/.irssi \
@@ -16,6 +13,7 @@ RUN set -x \
        && apk add --no-cache --virtual .build-deps \
                autoconf \
                automake \
+               ca-certificates \
                gcc \
                glib-dev \
                gnupg \
@@ -26,6 +24,7 @@ RUN set -x \
                ncurses-dev \
                openssl-dev \
                perl-dev \
+               perl-libwww \
                pkgconf \
        && wget "https://github.com/irssi/irssi/releases/download/${IRSSI_VERSION}/irssi-${IRSSI_VERSION}.tar.xz" -O /tmp/irssi.tar.xz \
        && wget "https://github.com/irssi/irssi/releases/download/${IRSSI_VERSION}/irssi-${IRSSI_VERSION}.tar.xz.asc" -O /tmp/irssi.tar.xz.asc \
@@ -53,7 +52,7 @@ RUN set -x \
                        | xargs -r apk info --installed \
                        | sort -u \
        )" \
-       && apk add --no-cache --virtual .irssi-rundeps $runDeps perl-libwww \
+       && apk add --virtual .irssi-rundeps $runDeps ca-certificates perl-libwww \
        && apk del .build-deps
 
 WORKDIR $HOME
````

The first `apk add --no-cache --virtual .build-deps ...` will fetch the APKINDEX and install it all. The second `apk add --virtual .irssi-rundeps ...` will only make the previously installed packages stick so they are not removed with the `apk del`. And the APKINDEX will only travel over the wire one single time instead of 3.